src/test/cli.up.test.ts

```diff
@@ -33,6 +33,15 @@ describe('Dev Containers CLI', function () {
 			await shellExec(`docker rm -f ${containerId}`);
 		});
 
+		it('should execute successfully with valid config and dotfiles', async () => {
+			const res = await shellExec(`${cli} up --workspace-folder ${__dirname}/configs/image --dotfiles-repository https://github.com/devcontainers/dotfiles`);
```

Comment:
This dotfiles repo is internal, I wonder if we should reference it here? Maybe you could create a public repo in https://github.com/codspace, our tests use that.

Reply:
The idea was to make that repo public after merging this pr.

Reply:
Moved to a different repo in that org.